zstacklib/zstacklib/utils/ovn.py

```
@@ -299,3 +306,214 @@ def addUplink(self, portPciMap, bondMode, lacpmode, ip, netmask, brName="br-phy"
+    @bash.in_bash
+    def setNicRxQueueConfig(self, nicName, queueNum, queueDescNum):
+        queueCmd = CtlBin + " --no-wait set Interface {} options:n_rxq={}".format(nicName, queueNum)
+        shell.run(queueCmd)
+        bufferCmd = CtlBin + " --no-wait set Interface {} options:rxq_desc={}".format(nicName, queueDescNum)
+        bash.bash_r(bufferCmd)
+
+    @bash.in_bash
+    def bindCpuCores(self, lMask, pmdMask):
+        # ovs-vsctl --no-wait set Open_vSwitch . other_config:dpdk-lcore-mask=0x3ff00
+        # ovs-vsctl --no-wait set Open_vSwitch . other_config:pmd-cpu-mask=0xff00
+        lMaskCmd = CtlBin + " --no-wait set Open_vSwitch . other_config:dpdk-lcore-mask=".format(lMask)
```

Ruff (0.11.9)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)